### PR TITLE
test: add workflow for weekly k8s tests

### PR DESCRIPTION
**Pull request by adeatcu-ionos**
## Checklist

- [x] PR name added as appropriate (e.g. `feat:`/`fix:`/`doc:`/`test:`/`refactor:`)
- [ ] Tests added or updated
- [ ] Documentation updated
- [ ] Changelog updated and version incremented (label: upcoming release)
- [ ] Github Issue linked if any
- [x] Jira task updated

### CODE REVIEW

**Review by avirtopeanu-ionos — APPROVED** [review]

File: .github/workflows/k8s-test-run.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+name: Weekly k8s test run
+on:
+  schedule:
+    # At 09:00:00 AM, every week on Monday, every month
+    - cron: '0 09 * * MON'
```
Comment (avirtopeanu-ionos) [inline .github/workflows/k8s-test-run.yml:5]: Just as a recommendation, this is in UTC and would mean 12 PM for Bucharest time. Nothing wrong with this, but I would consider perhaps moving this to a less active hours, such as 5-6 AM Bucharest time (2-3AM UTC)

Comment (adeatcu-ionos) [reply .github/workflows/k8s-test-run.yml]: @avirtopeanu-ionos thank you, I agree, I modified the time on this PR. Note that, for all other periodic workflows, we have the same hour: `09:00 AM`. I will note and maybe in the future we will also modify the hour for the rest of the periodic workflows.

Comment (adeatcu-ionos) [reply .github/workflows/k8s-test-run.yml]: @avirtopeanu-ionos @mimihalescu done in https://github.com/ionos-cloud/terraform-provider-ionoscloud/pull/329/commits/05a9a6921bc39169c2e17333f7b0e79d21710be2

File: .github/workflows/k8s-test-run.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+name: Weekly k8s test run
+on:
+  schedule:
+    # At 16:00:00 - UTC, every week on Monday, every month.
+    - cron: '0 16 * * MON'
```
Comment (cristiGuranIonos) [inline .github/workflows/k8s-test-run.yml:5]: let's put it earlier in the day(9-10 UTC)  - if it happens in the evening the on-call guys get notifications and it's not good.

**Pull request merged**